README.md

```diff
@@ -86,31 +97,46 @@ Make sure this is consistent with the redirect URI used in `okta_app_auth_config
 if your **Redirect URI** is `com.okta.example:/callback`, the **AppAuth Redirect Scheme** should be
 `com.okta.example`.
 
+#### Chrome Custom Tabs `ERR_UNKNOWN_URL_SCHEME`
+
+There is a [known issue](https://github.com/okta/okta-sdk-appauth-android/issues/8) when redirecting back to the a URI scheme from the browser via Chrome Custom Tabs. This is due to Chrome **not supporting** JavaScript initiated redirects back to native applications.
```

Comment:
nit: "to the a" -> "to a"

Also, is it really that Chrome doesn't support JS initiated redirects back to native apps in general, or do we need to be more explicit that it doesn't work in our case (where there's some async stuff happening).

Reply:
At the highest level - Chrome does not support redirects that were **not** triggered by the user.

As concise as that description is, I'm afraid it may imply that this "should work" given the user clicks "Login". Mentioning that it applies to only async JS redirects seems like it might be **too** much information. This was me trying to meet somewhere in the middle.